http-client/src/main/java/io/airlift/http/client/HttpClientModule.java

```diff
@@ -124,6 +127,8 @@ public void setInjector(Injector injector)
         @Override
         public HttpClient get()
         {
+            KerberosConfig kerberosConfig = injector.getInstance(Key.get(KerberosConfig.class));
```

Comment:
Oh, yeah, I had it with the annotation before and left it with Key.get when I got rid of that part.